dataflow/build.gradle

```diff
@@ -54,7 +54,7 @@ createDataflowShaded('shaded')
 createDataflowShaded('nullaway')
 createDataflowShaded('errorprone')
 
-task liveVariableTest(dependsOn: compileTestJava, group: 'Verification') {
+task liveVariableTest(dependsOn: ':checker:assemble', group: 'Verification') {
```

Comment:
This task must depend on `compileTestJava`.  You can add multiple dependOn in the body on the task.  But I don't think it needs `checker:assemble`; I don't see where checker.jar is used.  Why did you make this change?

Reply:
Thanks, I independently figured that out too.

I made the change because, as noted at the top of this pull request, running `./gradlew :dataflow:liveVariableTest` from a clean checkout does not work.  The failure is ClassNotFoundException for BugInCF.  Maybe some smaller dependence would also work; I did not try to minimize the dependency, but I agree it would be better to minimize it.